python_modules/dagster/dagster/_core/definitions/asset_graph.py

```diff
@@ -80,6 +82,7 @@ def __init__(
         code_versions_by_key: Mapping[AssetKey, Optional[str]],
         is_observable_by_key: Mapping[AssetKey, bool],
         auto_observe_interval_minutes_by_key: Mapping[AssetKey, Optional[float]],
+        required_assets_and_checks_by_key: Mapping[SpecKey, AbstractSet[SpecKey]],
```

Comment:
Can we get rid of `required_multi_asset_sets_by_key`?

Reply:
yup

Reply:
removed in https://github.com/dagster-io/dagster/pull/16928